packages/volto/src/components/manage/Widgets/ImageWidget.jsx

```diff
@@ -184,15 +195,29 @@ const UnconnectedImageInput = (props) => {
     >
       <Dropzone
         noClick
-        onDrop={handleUpload}
+        accept="image/*"
+        onDrop={(acceptedFiles) => {
+          if (acceptedFiles.length > 0) {
+            handleUpload(acceptedFiles);
+          } else {
+            setDragging(false);
+            toast.error(
+              <Toast
+                error
+                title={intl.formatMessage(messages.imageUploadErrorTitle)}
```

Comment:
I suggest to use the existing 'message.error' for the toast title. Not introducing new translation items without need.
See packages/volto/src/components/manage/Toolbar/More.jsx